src/Routes.jsx

```diff
@@ -73,6 +74,7 @@ const Routes = (props) => (
     <AuthenticatedRoute path="/admin_edit_user/:userId" component={AdminEditUser} props={props} rolesAllowed={[USER_ROLES.admin]} />
     <AuthenticatedRoute path="/manage_dac" component={ManageDac} props={props} rolesAllowed={[USER_ROLES.admin, USER_ROLES.chairperson]} />
     <AuthenticatedRoute path="/manage_edit_dac/:dacId" component={ManageEditDac} props={props} rolesAllowed={[USER_ROLES.admin, USER_ROLES.chairperson]} />
+    {checkEnv(envGroups.DEV) && <AuthenticatedRoute path="/manage_edit_dac_daa/:dacId" component={ManageEditDacDaa} props={props} rolesAllowed={[USER_ROLES.admin, USER_ROLES.chairperson]} />}
```

Comment:
For a future PR - given the sprawling nature of this feature, we could have a utility function for checking for this feature instead of checking for env, something like:
```suggestion
    {daaEnabled && <AuthenticatedRoute path="/manage_edit_dac_daa/:dacId" component={ManageEditDacDaa} props={props} rolesAllowed={[USER_ROLES.admin, USER_ROLES.chairperson]} />}
```
and define that variable somewhere (TBD):
```
    const daaEnabled = checkEnv(envGroups.DEV);
```